docs/multicluster/user-guide.md

```diff
@@ -424,72 +438,64 @@ spec:
+`default/antrea-mc-nginx` will be created in `test-cluster-east`, as well as
+a ServcieImport CR with name `default/nginx`. The Service `antrea-mc-nginx`
+contains the Endpoints exported from `test-cluster-west`, and any request to the
+Service from `test-cluster-east` will be routed to the backend `nginx` Pods in
+`test-cluster-west` through the Multi-cluster Gateways.
+
+As part of the Service export/import process, in the leader cluster, two
+ResourceExport CRs will be created in the Multi-cluster Controller Namespace,
+for the exported Service and Endpoints respectively, as well as two
+`ResourceImport` CRs. You can check them in the leader cluster with commands:
 
 ```sh
-$kubectl get resourceexport
+$kubectl get resourceexport -n antrea-multicluster
 NAME                                        AGE
 test-cluster-west-default-nginx-endpoints   7s
 test-cluster-west-default-nginx-service     7s
 
-$kubectl get resourceimport
+$kubectl get resourceimport -n antrea-multicluster
 NAME                      AGE
 default-nginx-endpoints   99s
 default-nginx-service     99s
 ```
 
-Then you can go to the member cluster `test-cluster-east` to check the new created
-Service and Endpoints with name `kube-system/antrea-mc-nginx` and a ServiceImport named
-`kube-system/nginx`. If there is already an existing Service created by users in the
-member cluster `test-cluster-east` also named `nginx` in Namespace `kube-system`, which
-should have no Antrea Multi-cluster annotation, then Multi-cluster Controller will
-simply skip the Service and Endpoints creation.
-
-If there is any new change from the exported Service or Endpoints, the derived multi-cluster
-resources will be updated accordingly. A few cases below are worth to note:
-
-1. When there is only one Service ResourceExport, Antrea Multi-cluster Controller will converge
-   the change and reflect the update in correspoding ResourceImport. Otherwise, controller will skip
-   converging the update until users correct it to match the Service definition in correspoding
-   ResourceImport.
-2. When a member cluster has already exported a Service, e.g.: `default/nginx` with TCP
-   Port `80`, then other member clusters can only export the same Service with the same Ports
-   definition including port names. Otherwise, Antrea Multi-cluster Controller will skip converting
-   the mismatched ResourceExport into the corresponding ResourceImport until users correct it.
-3. When a member cluster's Service ResourceExport has not been converged successfully
-   due to forementioned mismatch issue, Antrea Multi-cluster Controller will also skip converging
-   the corresponding Endpoints ResourceExport until users correct it.
+If there is any new change from the exported Service, the generated
+multi-cluster Service resources will be updated accordingly. A few cases below
+are worth to note:
+
+1. When there are more than one ServiceExports created for a single Service, the
```

Comment:
I am trying to revise the description (as hard to understand the previous version). Could you check?